episodes/04-redirection.md

```diff
@@ -236,6 +236,23 @@ $ wc -l SRR098026.fastq
 
 Now you can divide this number by four to get the number of sequences in your fastq file.
 
+This can be done using [shell integer arithmetic](https://www.gnu.org/software/bash/manual/html_node/Shell-Arithmetic.html)
+
+```bash
+$ echo $((996/4))
+```
+
+or [bc - an arbitrary precsion calculator](https://www.gnu.org/software/bc/manual/html_mono/bc.html)
```

Comment:
```suggestion
Note, this will do integer division - if you need floating point arithmetic you can use [bc - an arbitrary precision calculator](https://www.gnu.org/software/bc/manual/html_mono/bc.html)
```